build.gradle.kts

```diff
@@ -1,10 +1,11 @@
 import org.jetbrains.grammarkit.tasks.GenerateLexer
 import org.jetbrains.grammarkit.tasks.GenerateParser
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import com.github.rjeschke.txtmark.Processor
 
 plugins {
     idea
-    id("org.jetbrains.intellij") version "0.4.20"
+    id("org.jetbrains.intellij") version "0.4.21"
```

Comment:
Please not yet update this to the latest as for some reason the newest version introduces some bug where the plugin crashes. There is a upstream issue over at Jetbrains about it. It is only fixed in the next release of their IDE :/

Reply:
will do

Reply:
To be exact this update caused at least on my setup issue #13